src/lwc/gsChecklistItem/gsChecklistItem.html

```diff
@@ -0,0 +1,37 @@
+<template>
+    <div class="item-card">
+        <div class="card-image">
+            <img if:true={hasLink} src={imgSrc}/>
```

Comment:
If the image has meaning (i.e., the user needs to understand what that image _says_ in order to understand the page), then alternate text is required here (which may mean a new field is needed in the CMDT object). Otherwise, please specify null alternate text so that screen readers to not read out the file name. https://www.w3.org/WAI/tutorials/images/